### PR TITLE
amg departure check screen updates

### PR DESCRIPTION
**Pull request by ho-henry-senior**
**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

File: src/app/garfile/amg/checkin/pane.njk [hunk 1]
```diff
@@ -60,7 +60,7 @@
 <form action="/garfile/amg/checkin" method="post" class="form">
   <h2 class="govuk-heading-m">{{__('heading_submit_gar')}}</h2>
   <p class="govuk-body-m">{{__('submit_gar_caption')}}</p>
-  <input id='review-submit' type="submit" {% if garfile.status.name != 'StatusCheckComplete' %}disabled{% endif %} class="govuk-button" value="{{__('amg_check_form_submit')}}">
+  <input id='review-submit' type="submit" {% 'disabled' if garfile.status.name != 'StatusCheckComplete' %} class="govuk-button" value="{{__('amg_check_form_submit')}}">
```
Comment (ame-test) [inline src/app/garfile/amg/checkin/pane.njk:63]: Question: Is the `{% 'disabled' if garfile.status.name != 'StatusCheckComplete' %}` a valid **if expression**?

I presume it works.

- From the [docs](https://mozilla.github.io/nunjucks/templating.html#if-expression), it seems to be `{{ "true" if foo else "false" }}`.
- Also done in the departure status variant.

File: src/app/garfile/amg/departurestatus/pane.njk [hunk 2]
```diff
@@ -1,36 +1,46 @@
...
-{% set statustitle = 'This passenger is still being recorded ' %}
-{% else %}
-{% set statuscolour = 'red' %}
-{% set statustitle = 'There was a problem recording whether this passenger embarked with the craft ' %}
-{% endif %}
+  {% set status = person.amgDepartureStatus.name %}
 
+  {% if (status == 'Complete') %}
+    {% set statuscolour = 'green' %}
+    {% set statustitle = ['This passenger has been recorded as having ',
```
Comment (ame-test) [inline src/app/garfile/amg/departurestatus/pane.njk:7]: Question: Does the `+` string concatenation not work? e.g.
```handlebars
{% set 'This passenger has been recorded as having ' + '' if travelled else 
        'not ' + 'embarked with the craft' %}
 ```

**Review by ame-test — APPROVED** [review]
Looks good enough to me --> provide it has been tested to work.
* The questions are stylistic, relevantly unimportant.

File: src/app/garfile/amg/checkin/pane.njk (hunk 1 above)
Comment (ho-henry-senior) [reply src/app/garfile/amg/checkin/pane.njk:63]: yes it does work, and  I didn't want an else condition

**Pull request merged**